src/components/breadcrumb/breadcrumb.js

```diff
@@ -2,13 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Breadcrumb } from 'gatsby-plugin-breadcrumb';
 
-const mainSiteUrl = process.env.SITE_URL || 'https://ssw.com.au';
-
-const Breadcrumbs = ({ crumbs, crumbLabel }) => {
+const Breadcrumbs = ({ crumbs, crumbLabel, parentSiteUrl }) => {
```

Comment:
Hey @patriciabarrosssw I'd import the site config instead of having it passed in

Reply:
done